pyabc/acceptor/acceptor.py

```diff
@@ -350,14 +352,17 @@ def __init__(
             Whether to apply weights to correct for a bias induced by
             samples exceeding the density normalization. This may be False
             usually only for testing purposes.
+        log_file: str, optional
+            A log file for storing data of the acceptor that are currently not
+            saved in the database. The data are saved in json format.
```

Comment:
why did we choose the JSON format? Why not CSV?

Reply:
this ist just a bare-boned implementation which i needed quickly. I took json because it's human readable and easy to handle. If you think about a better, more general way of auto-saving all this stuff in the future, welcome.